v3/integrations/nrb3/go.mod

```
@@ -1 +1,5 @@
 module github.com/newrelic/go-agent/v3/integrations/nrb3
+
+go 1.20
```

```suggestion
go 1.20
```

does this need to be Golang version 1.20?

---

I don't know. nrb3 isn't part of the work done for this. nothing changed except the go.mod file, probably by a script while cleaning up go.mods. I don't have a problem moving it back to the lowest supported release.